src/decorators/restrict.js

```diff
@@ -1,9 +1,9 @@
 export default function restrict(func) {
 	return (target, key, descriptor) => {
 		const wrapped = descriptor.value
-		descriptor.value = function(...fnargs) {
-			if (func(fnargs[0])) {
-				return wrapped.apply(this, fnargs)
```

Comment:
You removed `return` keyword, does this still work?

Reply:
yep i have test with this restrict : 
```js
@restrict(({ member }) => member.roles.some(({ name }) => name == 'Admin'))
@restrict(({ member }) => member.roles.some(({ name }) => name == 'Test'))
```

and its works ...